sympy/profile.py

```diff
@@ -0,0 +1,23 @@
+import cProfile
+
+def profiling(fname):
+    """ Profile the functions.
+    Display its statistics.
+
+    Parameters
+    ==========
+
+    fname : string
+            It is the name of the function to be profiled.
+            Pass the parameter similar to how the function 
+            is invoked.
+            Example: str(<class_instance>.<function_name(<parameters>))
+
+    Returns
+    =======
+
+    void function : none type
+
+    """
+
+    cProfile.run(fname)
```

Comment:
We can just use `Profile.run` directly. I don't think adding a function in sympy that calls this indirectly is useful.

Reply:
I gave a try in profiling the slow functions in the class KanesMethod and LagrangeMethod. So as a first step, I thought of including a function for profiling but now I ignore it.